packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -82,6 +82,8 @@ if [ "$1" = "purge" ]; then
         umount -l "$mnt" || true
     done
     umount -l /run/snapd/ns/ || true
+    # remove all the current mount profiles that may exist.
+    rm -f /run/snapd/ns/snap.*.fstab
```

Comment:
What's the relationship between this and the line just above?  /run/snapd/ns is a tmpfs, isn't it?  Why are we removing things there after we umounted the tmpfs?